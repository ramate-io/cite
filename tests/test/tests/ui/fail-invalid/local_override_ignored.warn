warning: use of deprecated function `function_with_local_warn_level::_::_citation_warning`: Citation requires documentation but no annotation provided. Add a 'reason = "..."' attribute or enable the annotationless feature
 --> tests/ui/fail-invalid-warn/local_override_ignored.rs:5:1
  |
5 | #[cite(mock, same = "content", annotation = "ANY")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(deprecated)]` on by default
  = note: this warning originates in the attribute macro `cite` (in Nightly builds, run with -Z macro-backtrace for more info)
