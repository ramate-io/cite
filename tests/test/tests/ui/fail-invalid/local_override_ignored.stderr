error: Citation requires documentation but no annotation provided. Add a 'reason = "..."' attribute or enable the annotationless feature
 --> tests/ui/fail-invalid/local_override_ignored.rs:5:1
  |
5 | #[cite(mock, same = "content", annotation = "ANY")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `cite` (in Nightly builds, run with -Z macro-backtrace for more info)
